plugins/amzn/amzn-nvme.c

```diff
@@ -253,9 +257,11 @@ static int get_stats(int argc, char **argv, struct command *cmd,
 		return -ENOTSUP;
 	}
 
+#ifdef CONFIG_JSONC
 	if (!strcmp(cfg.output_format, "json"))
 		amzn_print_json_stats(&log);
 	else
```

Comment:
Thanks for adding validate_output_format. Instead parsing the `cfg.output_format` again you could use the `flags` variable directly 

```
if (flag & JSON`)
  amzn_print_json_stats
```

BTW, I would prefer to introduce an empty `amzn_prin-json_stats` implementation in case json-c is missing. This avoid sprinkling the `ifdefs` around in the code. IMO, this helps maintaining the code base in the long run.

Reply:
Yes I agree with you so just fixed this by the PR #2636. Thank you.